src/subscription/UpgradeSubscriptionWizard.ts

```diff
@@ -188,15 +188,9 @@ export async function loadSignupWizard(subscriptionParameters: SubscriptionParam
 			await logins.logout(false)
 		}
 
-		if (signupData.newAccountData) {
-			m.route.set("/login", {
-				loginWith: signupData.newAccountData.mailAddress,
-			})
-		} else {
-			m.route.set("/login", {
-				noAutoLogin: true,
-			})
-		}
+		m.route.set("/login", {
+			noAutoLogin: true,
```

Comment:
I think in this case it would be better to add `noAutoLogin: true` to the case with `newAccountData` so we get to the login form if the account was created (so we keep the branch and pass both `loginWith` and `noAutoLogin` to the `m.route` call)

For example, select a free account in signup, select mail address, wait for it to be created and the recovery code is shown and then cancel out of the wizard until you are back to the login screen -> it should show the form with the new address pre-filled to indicate that the account was created.

make sure to review the test notes after this is changed